plugin/trino-http-event-listener/src/main/java/io/trino/plugin/httpquery/HttpEventListenerConfig.java

```diff
@@ -127,4 +131,45 @@ public int getRetryCount()
     {
         return retryCount;
     }
+
+    @ConfigDescription("Delay in seconds between retries")
+    @Config("http-event-listener.connect-retry-delay")
+    public HttpEventListenerConfig setRetryDelay(Duration retryDelay)
+    {
+        this.retryDelay = retryDelay;
+        return this;
+    }
+
+    public Duration getRetryDelay()
+    {
+        return this.retryDelay;
+    }
+
+    @ConfigDescription("Base used for exponential backoff when retying on server error. Formula is attemptDelay = retryDelay * backoffBase ^ attemptCount. " +
+            "Attempt counting starts from 0. Leave empty or set to 1 to disable.")
+    @Config("http-event-listener.connect-backoff-base")
+    public HttpEventListenerConfig setBackoffBase(double backoffBase)
+    {
+        this.backoffBase = backoffBase;
+        return this;
+    }
+
+    @Min(value = 1, message = "Exponential base must be a positive, non-zero integer.")
+    public double getBackoffBase()
+    {
+        return this.backoffBase;
+    }
+
+    @ConfigDescription("Maximum delay between retries. This should be used with exponential backoff.")
+    @Config("http-event-listener.connect-max-delay")
+    public HttpEventListenerConfig setMaxDelay(Duration maxDelay)
+    {
+        this.maxDelay = maxDelay;
```

Comment:
nit:
would be nice to check if this is >= retryDelay. But I am not sure if that is easy to cross check different configuration options. @findepi do you happen to know?